File: pset1/mario/less/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;
    // Get user input and re-check if n is outside 1-22 range
    do
    {
        height = get_int("Height:");
    }
    while (height < 0 || height > 23);
    // Build half pyramid of '#' where height is user input
    for (int row = 0; row < height; row++)
    {
        for (int space = height - row - 1; space > 0; space--)
        {
            printf(" ");
        }
        for (int hash = row + 2; hash > 0; hash--)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/mario/less/mario.py
from cs50 import get_int

# Get user input and reprompt if input is outside 1 - 22 range
while True:
    height = get_int("Height:")
    if height < 24 and height > -1:
        break

# Build half pyramid of '#' where height is user input
for i in range(height):
    print(" " * (height - i - 1), end="")
    print("#" * (i + 2), end="")
    print()